core/services/job/kv_orm.go

```diff
@@ -0,0 +1,69 @@
+package job
+
+import (
+	"encoding/json"
+	"fmt"
+	"time"
+
+	"github.com/jmoiron/sqlx"
+	"github.com/jmoiron/sqlx/types"
+
+	"github.com/smartcontractkit/chainlink/v2/core/logger"
+	"github.com/smartcontractkit/chainlink/v2/core/services/pg"
+)
+
+// KVStore is a simple KV store that can store and retrieve serializable data.
+//
+//go:generate mockery --quiet --name KVStore --output ./mocks/ --case=underscore
+type KVStore interface {
+	Store(key string, val interface{}) error
+	Get(key string, dest interface{}) error
+}
+
+type jobKVStore struct {
```

Comment:
Just `kvStore` should be fine, since we are already inside of `package job`
```suggestion
type kvStore struct {
```